conversion/conversion.go

```diff
@@ -77,19 +74,22 @@ const (
 	// This is an experimental driver; implementation in progress.
 	DYNAMODB string = "dynamodb"
 
-	// Target db for which schema is being generated.
-	TARGET_SPANNER               string = "spanner"
-	TARGET_EXPERIMENTAL_POSTGRES string = "experimental_postgres"
+	// TargetSpanner is the target db for which schema is being generated.
+	TargetSpanner string = "spanner"
+
+	// TargetExperimentalPostgres is the target for experimental postgres
```

Comment:
Go comments are usually recommended to be complete sentences that end with a period.
https://golang.org/doc/effective_go#commentary

There are '.' missing in multiple other places as well.

Reply:
I see, I'll add the periods in!